browser/src/react-redux/App/Login/Login.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react'
+
+export default class Login extends React.Component {
+  constructor(props) {
+    super(props)
```

Comment:
You don't even need `props` in the constructor. Let's get rid of these parameters. If you do need props in the constructor, probably, the only time you will need to pass it to `super()` is when you need access to `this.props` or are invoking functions that grab props in that way.